docs/_articles/en/v3.x/scanner-commands/run.md

```diff
@@ -80,84 +80,81 @@ Returns retire-js violation messages details about each vulnerability, including
 ```
 
 This example evaluates rules against somefile.js, including Jasmine in the environment variables. Uses --env to override the default ESLint environment variables to add frameworks.
 
 ```bash
-$ sfdx scanner:run --target "somefile.js" --env '{"jasmine": true}'
+$ sf scanner run --target "somefile.js" --env '{"jasmine": true}'
 ```
 
 This example evaluates rules aginst somefile.js using eslint-lwc and pmd engines. Use --engine to include or exclude engines. Any engine listed will be run, regardless of its current 'disabled' attribute.
 
 ```bash
-$ sfdx scanner:run --target "somefile.js" --engine "eslint-lwc,pmd"
+$ sf scanner run --target "somefile.js" --engine "eslint-lwc,pmd"
 ```
 
 In this example, ESLint and RetireJS will run even if they’re disabled, and no other engines will run. Use --engine to include or exclude engines. Regardless of their current ‘disabled’ attribute, any specified engine will run, and all others will not. 
 ```bash
-$ sfdx scanner:run --target "somedirectory" --engine "eslint,retire-js"
+$ sf scanner run --target "somedirectory" --engine "eslint,retire-js"
 ```
 
 Use --engine to invoke engines that are not enabled by default.
 This example executes CPD engine against known file extensions in "/some/dir". CPD helps detect blocks of code duplication in selected languages.
 
 ```bash
-$ sfdx scanner:run --target "/some/dir" --engine cpd
+$ sf scanner run --target "/some/dir" --engine cpd
  ```
  
 This example executes rules defined in pmd_rule_ref.xml against the files in 'src'. To use PMD with your own rule reference file, use --pmdconfig. Note that rule filters are not applied.
 
 ```bash
-$ sfdx scanner:run --target "src" --pmdconfig "pmd_rule_ref.xml"
+$ sf scanner run --target "src" --pmdconfig "pmd_rule_ref.xml"
 ```
 
 This example uses a custom config to scan the files in 'src'. To use ESLint with your own .eslintrc.json file, use --eslintconfig. Make sure that the directory you run the command from has all the NPM dependencies installed.
 
 ```bash
-$ sfdx scanner:run --target "src" --eslintconfig "/home/my/setup/.eslintrc.json"
+$ sf scanner run --target "src" --eslintconfig "/home/my/setup/.eslintrc.json"
 ```
 
 This example uses --normalize-severity to output normalized severity and engine-specific severity across all engines. Normalized severity is: 1 (high), 2 (moderate), and 3 (low). 
 
 ```bash
-$ sfdx scanner:run --target "/some-project/" --format csv --normalize-severity
+$ sf scanner run --target "/some-project/" --format csv --normalize-severity
 ```
 
 This example uses --severity-threshold to throw a non-zero exit code when rule violations of normalized severity 2 or greater are found. If any violations with the specified severity (or greater) are found, the exit code equals the severity of the most severe violation.
 
 ```bash
-$ sfdx scanner:run --target "/some-project/" --severity-threshold 2
+$ sf scanner run --target "/some-project/" --severity-threshold 2
 ```
-
-## Demo
```

Comment:
@jfeingold35 at this point, I think we should just remove the outdated Demo gif, and when we have more time we can consider developing a more robust video if we want to.

Reply:
@teresa-allen-sfdc , should we maybe remove the demo videos from all of the commands, then?

Reply:
Yes @jfeingold35